scripts/pd.json

```diff
@@ -58,8 +58,10 @@
   "gnetId": null,
   "graphTooltip": 1,
   "id": null,
-  "iteration": 1564738811479,
-  "links": [],
+  "iteration": 1566379062374,
+  "links": [
```

Comment:
Why there is a lot of changes like this?

Reply:
Sorry that's my fault. I will use `jq` to reformat this json file.

Reply:
Fixed